client/src/components/ratingsReviews/NewReview.jsx

```diff
@@ -134,7 +135,7 @@ class NewReview extends React.Component {
           [id]: e.target.value
         });
       }
-    } else if (id === 'Characteristic') {
+    } else if (name === 'Size' || name === 'Comfort' || name === 'Quality' || name === 'Length' || name === 'Fit' || name === 'Width') {
       this.characteristics[currentCharacteristic.id] = parseInt(e.target.value);
```

Comment:
I've noticed that you use `parseInt` a lot. Is that some sort of failsafe? It seems like most of the things you're using it on should already be numbers.

Reply:
In this case its not a fail safe, e.target.value returns a string.

Reply:
I never even realized the ids were strings :)